src/cloud-code/Parse.Cloud.js

```diff
@@ -528,6 +529,36 @@ ParseCloud.beforeConnect = function (handler, validationHandler) {
   );
 };
 
+/**
+ * Sends an email through the Parse Server mail adapter.
+ *
+ * **Available in Cloud Code only.**
+ * **Requires a mail adapter to be configured for Parse Server.**
+ *
+ * ```
+ * Parse.Cloud.sendMail({
+ *   from: 'Example <test@example.com>',
+ *   to: 'contact@example.com',
+ *   subject: 'Test email.',
+ *   text: 'This email is a test'
+ * });
+ *```
+ *
+ * @method sendMail
+ * @name Parse.Cloud.sendMail
+ * @param {Object} data The object of the mail data to send
+ */
+ParseCloud.sendMail = function (data) {
+  const config = Config.get(Parse.applicationId) || {};
+  const emailAdapter = config.userController.adapter;
+  if (emailAdapter) {
```

Comment:
I think the code style you had previously was better practice; the main flow should be the non-error flow, while failing conditions jump out of the main flow and return earlier:

```js
if (!emailAdapter) {
    config.loggerController.warn(  //still open whether this shouldn't be error rather than warn
        'Failed to send email because no mail adapter is configured for Parse Server.'
    );
    return;
}
return emailAdapter.sendMail(data);   
```